src/app/_components/userProfile/UserProfile.tsx

```diff
@@ -1,18 +1,34 @@
-"use client";
+import QRCode from "react-qr-code"
+import { type Session } from "next-auth";
+import { type Team } from "@prisma/client";
+import SignOut from "~/app/_components/userProfile/signOut";
 
-import { Session } from "next-auth";
-import { api } from "~/trpc/react";
-
-interface UserProfileProps {
```

Comment:
My personal preference is that we keep this interface for code clarity

Reply:
The reason I advocate for inline interfaces is that right now we don't have any re-use cases. Using inline interfaces reduces decision fatigue, keeps concerns separated, and keeps the data structures small.

Reply:
Yea, that's fair, I understand your point. In my eyes, these interfaces wouldn't cause decision fatigue since they'd just be a standard that we have every time, and they'd help cleanup the code for function signatures, which can get messy without them. Could you elaborate on how it keeps data structures small?

Reply:
Forgot what was on my mind at the time I wrote the comment but I'll let you know once it comes back to me.